python/lsst/obs/base/cli/cmd/register_instrument.py

```diff
@@ -30,7 +30,7 @@
 log = logging.getLogger(__name__)
 
 
-@click.command("register_instrument")
+@click.command()
```

Comment:
In daf_butler I see 
```
@click.command(name="config-validate")
```
and I also see the form above. Neither of these is the form stated on the ticket as the plan so I'm more confused than when I started 😄